workshop/exercises/pygeoapi.config.yml

```diff
@@ -184,7 +184,44 @@ resources:
               data: tests/data/ne_110m_admin_0_countries.sqlite
               id_field: ogc_fid
               table: ne_110m_admin_0_countries
-
```

Comment:
Please leave a blank line between this and the previous collection (as in the other examples).